packages-next/types/src/legacy-filters.ts

```diff
@@ -3,7 +3,7 @@ import { schema } from '.';
 const identity = (x: any) => x;
 
 export const impls = {
-  equalityConditions<T>(fieldKey: string, f: (a: any) => any = identity) {
+  equalityConditions<T>(fieldKey: string, f: (a: any) => any = identity): any {
```

Comment:
These things are because TypeScript generates declarations that are broken but having actual types for the returns types here doesn't matter.